app/models/exporter.rb

```diff
@@ -0,0 +1,31 @@
+module Exporter
+
+  class << self
+    attr_accessor :time
+
+    def initialize(time=Time.now)
```

Comment:
Surrounding space missing in default value assignment.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.